interface/billing/sl_receipts_report.php

```diff
@@ -133,12 +138,12 @@ function sel_diagnosis() {
 
 </script>
 
-<title><?php xl('Cash Receipts by Provider','e')?></title>
+<title><?php echo xlt('Cash Receipts by Provider')?></title>
 </head>
 
 <body class="body_top">
 
-<span class='title'><?php xl('Report','e'); ?> - <?php xl('Cash Receipts by Provider','e'); ?></span>
+<span class='title'><?php echo xlt('Report'); ?> - <?php echo xlt('Cash Receipts by Provider'); ?></span>
```

Comment:
I guess this separation is just for the translation engine.  Seems like there is a better option, like running a translation matrix against contiguous names of reports and features.